R/zzz.R

```diff
@@ -44,6 +44,45 @@ default_linters <- modify_defaults(
 #' There is a list for the default elements and another that contains all available elements.
 #' Use [modify_defaults()] to produce a custom list.
 #'
+#' @details
+#' The following functions are sometimes regarded as undesirable:
+#'
+#'  * `attach()` modifies the global search path. Use roxygen2's @importFrom statement in packages, or `::` in scripts.
```

Comment:
should these all be links? `[attach()]`, etc?

Reply:
I thought about that too, wasn't sure because `withr` e.g. is just Suggested and not Imported, therefor not all links are guaranteed to work.

Reply:
I _think_ we're allowed to link Suggests. Since worst case the link is just dead -- doesn't affect the code running or anything. But I can't see any documentation as of now.